suite-common/wallet-utils/src/transactionUtils.ts

```diff
@@ -267,6 +268,9 @@ export const analyzeTransactions = (
         });
     }
 
+    // make sure the known transactions are sorted properly
+    const knownSorted = known.slice(0).sort(sortByBlockHeight);
```

Comment:
i totally forgot that there may be null values, so in that case it should be excluded from the list
```
known.filter(tx => tx != null).sort(sortByBlockHeight);
```

Reply:
feda7ed86ad0ec0c23565670970ec0d957c1186d